juju/client/connection.py

```diff
@@ -36,13 +37,14 @@ class Connection:
     """
     def __init__(
             self, endpoint, uuid, username, password, cacert=None,
-            macaroons=None):
+            macaroons=None, loop=None):
```

Comment:
Doe sthis need some help text and some usage docs updates? It looks like this changes how this can be fundamentally be used.